stl/inc/bit

```diff
@@ -82,8 +82,8 @@ _NODISCARD constexpr _Ty bit_floor(const _Ty _Val) noexcept {
 }
 
 template <class _Ty, enable_if_t<_Is_standard_unsigned_integer<_Ty>, int> = 0>
-_NODISCARD constexpr _Ty bit_width(const _Ty _Val) noexcept {
-    return static_cast<_Ty>(numeric_limits<_Ty>::digits - _STD countl_zero(_Val));
+_NODISCARD constexpr int bit_width(const _Ty _Val) noexcept {
+    return numeric_limits<_Ty>::digits - _STD countl_zero(_Val);
 }
 

```

Comment:
There's a cast in `<format>` that can be dropped now:

https://github.com/microsoft/STL/blob/04ee87830a404437f1691d6a3e8809e4986d8493/stl/inc/format#L2942